src/main/kotlin/com/dansplugins/factionsystem/command/faction/info/MfFactionInfoCommand.kt

```diff
@@ -0,0 +1,47 @@
+package com.dansplugins.factionsystem.command.faction.info
+
+import com.dansplugins.factionsystem.MedievalFactions
+import com.dansplugins.factionsystem.player.MfPlayer
+import dev.forkhandles.result4k.onFailure
+import org.bukkit.ChatColor.RED
+import org.bukkit.ChatColor.AQUA
+import org.bukkit.command.Command
+import org.bukkit.command.CommandExecutor
+import org.bukkit.command.CommandSender
+import org.bukkit.entity.Player
+import java.util.logging.Level.SEVERE
+
+class MfFactionInfoCommand(private val plugin: MedievalFactions) : CommandExecutor {
+    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<out String>): Boolean {
+        if (!sender.hasPermission("mf.info")) {
+            sender.sendMessage("$RED${plugin.language["CommandFactionInfoNoPermission"]}")
+            return true
+        }
+        
+        if (sender !is Player) {
+            sender.sendMessage("$RED${plugin.language["CommandFactionInfoNotAPlayer"]}")
+            return true
+        }
+        plugin.server.scheduler.runTaskAsynchronously(plugin, Runnable {
+            // get player's faction
+            val playerService = plugin.services.playerService
+            val mfPlayer = playerService.getPlayer(sender)
+                ?: playerService.save(MfPlayer.fromBukkit(sender)).onFailure {
+                    sender.sendMessage("$RED${plugin.language["CommandFactionInfoFailedToSavePlayer"]}")
+                    plugin.logger.log(SEVERE, "Failed to save player: ${it.reason.message}", it.reason.cause)
+                    return@Runnable
+                }
+            val factionService = plugin.services.factionService
+            val faction = factionService.getFaction(mfPlayer.id)
+            if (faction == null) {
+                sender.sendMessage("$RED${plugin.language["CommandFactionInfoMustBeInAFaction"]}")
+                return@Runnable
+            }
+            // send player faction info
+            sender.sendMessage("$AQUA${plugin.language["CommandFactionInfoTitle", faction.name]}")
+            sender.sendMessage("$AQUA${plugin.language["CommandFactionInfoDescription", faction.description]}")
+            sender.sendMessage("$AQUA${plugin.language["CommandFactionInfoMembers", "" + faction.members.size]}")
```

Comment:
```suggestion
            sender.sendMessage("$AQUA${plugin.language["CommandFactionInfoMembers", faction.members.size.toString()]}")
```

Reply:
It might also be useful to do a role breakdown here

Reply:
What do you mean by breakdown here?

Reply:
e.g.
Members: 10
Roles:
  Owner: 2
  Officer: 3
  Member: 5

Reply:
I think this might get pretty long for some factions. We ought to have a 'roles' command instead of including this in the output of the info command.

Reply:
I created #1505 for this.